src/30_Advanced/Advanced_Video_Docking.html

```diff
@@ -0,0 +1,223 @@
+<!---{
+  "hidePreview": true,
+  "experiments": ["video-dock"]
+}--->
```

Comment:
Can you please add a `introduction` section that shortly explains what this samples is about

Reply:
Done.